doc/api/next_api_changes/removals/22107-OG.rst

```diff
@@ -0,0 +1,44 @@
+Removal of deprecated ``mathtext`` APIs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following :mod:`.mathtext` APIs have been removed:
+``MathtextBackendBitmap``,
+``MathtextBackendPs``,
+``MathtextBackendPdf``,
+``MathtextBackendSvg``,
+``MathtextBackendCairo``,
+``ship``,
+``Fonts``,
+``TruetypeFonts``,
+``StandardPsFonts``,
```

Comment:
Missing all subclasses of these: `BakomaFonts`, `UnicodeFonts`, `DejaVuFonts`, `DejaVuSerifFonts`, `DejaVuSansFonts`, `StixFonts`, `StixSansFonts`.

Reply:
See comment regarding `Accent`. Same here.